plugins/woocommerce-blocks/tests/e2e/tests/local-pickup/local-pickup.merchant.block_theme.side_effects.spec.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import { REGULAR_PRICED_PRODUCT_NAME } from '../checkout/constants';
+
+test.describe( 'Merchant → Local Pickup Settings', () => {
+	test.beforeEach( async ( { localPickupUtils } ) => {
+		await localPickupUtils.deleteLocations();
+		await localPickupUtils.addPickupLocation( {
+			location: {
+				name: 'Automattic, Inc.',
+				address: '60 29th Street, Suite 343',
+				city: 'San Francisco',
+				postcode: '94110',
+				state: 'US:CA',
+				details: 'American entity',
+			},
+		} );
+		await localPickupUtils.disableLocalPickupCosts();
+		await localPickupUtils.enableLocalPickup();
+	} );
+
+	test.afterEach( async ( { localPickupUtils } ) => {
+		await localPickupUtils.deleteLocations();
+		await localPickupUtils.disableLocalPickupCosts();
+		await localPickupUtils.setTitle( 'Local Pickup' );
+		await localPickupUtils.enableLocalPickup();
```

Comment:
We basically want to reset the local pickup to how it was before we saw it, but since we need a page object to do so, we can't do this in `beforeAll` or `afterAll`, therefore I defaulted to `beforeEach` and `afterEach` where page contexts exist